dns-dodo.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=The dns-dodo DNS record updater
+Requires=network-online.target
+After=network-online.target
+
+[Service]
+ExecStart=/usr/bin/dns-dodo update-dns -p
```

Comment:
After some research, I don't think providing a systemd unit file that is compatible with `go install` is all that practical. Part of reason is that while you can _specify_ environment variables within systemd unit files, you can't use them to construct relative paths.

We could get around this by including a launch script which uses an environment variable set in the unit (`GOPATH` in this case), but this still doesn't solve much - the user would still have to manually set their `GOPATH` in the unit and copy/link it to `~/.local/share/systemd/user`.

We could think about providing a unit template (or putting one to the README), but I think the existing unit file pointing at `/usr/bin` for an AUR build should be included nonetheless.